wallet/service/openapi.json

```
@@ -0,0 +1,95 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "Wallet API",
+    "description": "The version 2 of the wallet HTTP API leverages the wallet JSON-RPC API.\n\nSome useful links:\n- [Documentation of the wallet JSON-RPC API](https://docs.vega.xyz/docs/mainnet/api/jsonrpc/wallet)\n- [JSON-RPC 2.0 Specification](https://www.jsonrpc.org/specification)",
+    "version": "2.0.0"
+  },
+  "externalDocs": {
+    "description": "Find out more about Vega REST APIs.",
+    "url": "https://docs.vega.xyz/docs/mainnet/api/rest/overview"
+  },
+  "paths": {
+    "/v2/requests": {
+      "post": {
+        "summary": "Submit a JSON-RPC request to the wallet API.",
+        "description": "This endpoint is the main entrypoint to submit the JSON-RPC requests. The body of the request should be formatted as a JSON-RPC v2 request. The response is formatted as the JSON-RPC v2 response.\n\nBe sure to specify the `id` if you are interested in the response. See [the JSON-RPC notification documentation](https://www.jsonrpc.org/specification#notification) for more details.",
+        "requestBody": {
+          "description": "The JSON-RPC v2 request bidy",
```

```suggestion
          "description": "The JSON-RPC v2 request body",
```
Is the body the same as the description, or is there another way that this can be described to supplement the fact that it's called the "request body"? Repeating the word body doesn't help describe it much